src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/VirtualMachinesOverviewTabDetailsConsoleConnect.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { PlayIcon } from '@patternfly/react-icons';
+
+type VirtualMachinesOverviewTabDetailsConsoleConnectProps = {
+  connect?: () => void;
+};
+
+const VirtualMachinesOverviewTabDetailsConsoleConnect: React.FC<
+  VirtualMachinesOverviewTabDetailsConsoleConnectProps
+> = ({ connect }) => {
+  return (
+    <div className="vnc-grey-background">
+      <PlayIcon color="blue" size="md" onClick={connect} />
```

Comment:
a. color should be from patternfly pallern
b. when vm is off, it should be disabled ( gray ? )

Reply:
replaced with css var and added disabled apperance